cents-of-unity/src/App.tsx

```diff
@@ -1,29 +1,16 @@
 import React, { FC } from 'react'
-import { Layout, Menu } from 'antd'
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
 import './App.css'
-import Profile from './components/profile/Profile'
+import Home from './home/Home'
 
-const { Header, Content, Footer } = Layout
 const App: FC = () => (
   <div className="App">
-    <Layout className="layout">
-      <Header>
-        <div className="logo" />
-        <Menu theme="dark" mode="horizontal" defaultSelectedKeys={['2']}>
-          <Menu.Item key="1">nav 1</Menu.Item>
-          <Menu.Item key="2">nav 2</Menu.Item>
-          <Menu.Item key="3">nav 3</Menu.Item>
-        </Menu>
-      </Header>
-      <Content style={{ padding: '0 50px' }}>
-        <Profile
-          name="Bob Smith"
-          email="bob3@gatech.edu"
-          phoneNumber="123-456-7890"
-        />
-      </Content>
-      <Footer style={{ textAlign: 'center' }}>Cents of Unity Fall 2020</Footer>
-    </Layout>
+    <Router>
+      <Switch>
+        <Route exact path='/' component={Home}/>
+        <Route exact path='/profile' component={Home} />
+      </Switch>
+    </Router>
   </div>
 )
 export default App
```

Comment:
nit: newline at EOF

Reply:
now that my style PR is merged - you should be able to rebase off master and pre-commit will autofix that